common/components/AddStreetModal/index.tsx

```diff
@@ -44,9 +42,21 @@ const AddStreetModal: FC<Props> = ({
     }
   }
 
+  const getTitle = () => {
+    if (edit) return 'Редагування адреси'
+    if (preview) return 'Перегляд адреси'
+    return 'Додавання адреси'
+  }
+
+  const editable = preview || edit
```

Comment:
Huh? The form is editable only in `edit` mode. Preview is for preview. So...

```
<AddStreetForm form={form} editable={edit} />
```

If `edit` does not represent real state of form mode - fix it, plese